jobs/furnishings/src/furnishings/stage_processors/post_processor.py

```diff
@@ -77,7 +77,10 @@ def _update_furnishings_status(self, furnishing_group_id):
                 furnishing.save()
 
     def process(self):
-        """Postprocess to generate and upload file to external resources(BC Laws)."""
+        """Postprocess to generate and upload file to external resources (BC Laws)."""
+        if not self._furnishings_dict:
+            return
```

Comment:
In some cases, the furnishings job can be run but none of the stage processors run (for example, none of the stage cron strings can run that day). This results in an empty `xml_furnishings_dict`. Adding this check prevents an `xml_payload` entry from being created with an empty template.